pkg/apis/integreatly/v1alpha1/rhmi_types.go

```diff
@@ -50,22 +50,22 @@ var (
 	ProductCloudResources      ProductName = "cloud-resources"
 	ProductDataSync            ProductName = "datasync"
 
-	// PBrookes 08/08/2019:
 	// Could not find a way to determine these versions dynamically, so they are hard-coded
 	// It is preferable to determine the version of a product dynamically (from a CR, or configmap, etc)
+	// Follow up Jira: https://issues.redhat.com/browse/INTLY-5946
 	VersionAMQOnline           ProductVersion = "1.3.1"
-	VersionApicurito           ProductVersion = "1.0.1"
+	VersionApicurito           ProductVersion = "7.5"
 	VersionAMQStreams          ProductVersion = "1.1.0"
 	VersionCodeReadyWorkspaces ProductVersion = "2.0.0.GA"
 	VersionFuseOnOpenshift     ProductVersion = "master"
 	VersionMonitoring          ProductVersion = "1.1.4"
 	Version3Scale              ProductVersion = "2.7"
 	VersionUps                 ProductVersion = "2.3.2"
```

Comment:
Is this the correct product version of UPS that we install? I looked at the UPS console but I couldn't find info of what version it is from there. The manifest file we use for this only has has reference to `0.5.0` but not `2.3.2`.

maybe @secondsun @grdryn can help with this?

Reply:
The version of the operator is 0.5.0, and the version of UPS itself is \<checks notes> 2.3.2: https://github.com/aerogear/unifiedpush-operator/blob/master/pkg/constants/constants.go#L4

Reply:
I presume that the integreatly-operator needs this to be able to display it somewhere or something?

Reply:
Yeah, it's displayed in the solution explorer :smiley: . Thanks for the confirmation @grdryn! :+1:

Reply:
I think it's taken from here: https://github.com/aerogear/unifiedpush-operator/blob/f8b3254fed3ad63a8a2e8a75b5b2b45c09d5c8ab/pkg/constants/constants.go#L4 

Let's see if they can confirm if this is correct!